helm/g8s-oauth2-proxy-chart/templates/deployment.yaml

```diff
@@ -16,14 +16,18 @@ spec:
         k8s-app: oauth2-proxy
     spec:
       containers:
-        # Use bitly fork from https://github.com/giantswarm/oauth2_proxy
-      - image: quay.io/giantswarm/oauth2_proxy:38072cbaa6abfc6e9f647c1ee0e4c7afa5ba194e
+        # Use bitly fork from https://github.com/pusher/oauth2_proxy/tree/kubernetes,
```

Comment:
Comment confuses me a bit. Is this pusher repo a fork of the bitly repo? If so can you make that clearer.